### ISSUE TITLE
OWF TBL box weergave voorbeeld lijkt op Wikipedia

### ISSUE THREAD

**Issue by LloydRutledge** [ABI-Team-30/Fresnel-Forms#44]
Dit is een detail uit #27. Ik maak deze en binnenkort nog enkele details ervan als aparte issues om apart te volgen. Elke is relevant voor een specifiek deel van de Forms paper.

De issue is dat onze OWF TBL box weergave hetzelfde moet lijken als die op Wikipedia. Wij gaan deze tonen (via screendumps) en opschrijven in de Forms submission. Het is een kern deel van het ondersteunen van onze stelling dat het OWF-aanpak op WP toegepast kan worden.

De doel weergave is op  http://en.wikipedia.org/wiki/Tim_Berners-Lee . De laatste afbeelding is:

![knipsel](https://cloud.githubusercontent.com/assets/1098321/5940490/7ece552e-a70b-11e4-996a-61b6b25b49d8.JPG)

Die is wat simpeler geworden in de afgelopen weken!

Joop maakt een wiki voorbeeld op http://abiteam30.lukylx.org:3080/mediawiki/index.php/TimTest . Deze wikipagina laat alleen zien wat de wikicode van de fresne2wiki conversie moet worden. De conversie zelf wordt nog geïmplementeerd. Klopt dat, Joop, of is deze van de conversie? Heb je een manier om ook de conversie resultaat voor de TBL-pagina te laten zien? Misschien op dezelfde wiki? Die wordt dan een figure in de Forms submission.

Op http://is.cs.ou.nl/SW_CRS/index.php5/Wikipedia_Tim_Berners-Lee_scenario onderhoud ik nog meer informatie over deze voorbeeld.

**Comment by jheijning** [comment]
TimTest is niet van conversie. Ik heb de default CSS voor tabellen in fresnel2wiki nu wel aangepast naar de TBL weergave. Een label verbergen en daarbij de waarde in het midden zetten werkt al met de conversie, en ook CSS. Ondersteuning van afbeeldingen en custom delimiters komen er aan.

Dus nog een work in progress.

**Comment by jheijning** [comment]
http://abiteam30.lukylx.org:3080/mediawiki/index.php/TimTestConv
Deze is van conversie. De juiste ontologie is nog niet gebruikt (aanpassing van de Firm ontology). Ziet er nog niet goed uit, verbeteringen zijn onderweg.

**Comment by LloydRutledge** [comment]
Een verbetering is data data property waarden niet meer als links tonen. En vast nog meer natuurlijk.

**Comment by LloydRutledge** [comment]
Plaatje zit nu als Fig. 6 in Forms submission. Die blijven we actueel houden.

**Comment by jheijning** [comment]
@LloydRutledge  Wil je de honorificSuffix property http://mappings.dbpedia.org/index.php/OntologyProperty:HonorificSuffix ook nog in de ontologie hebben, staat er nu niet in.

**Comment by LloydRutledge** [comment]
Ja eigenlijk. Heel knap dat je dat vond. DBPedia export TBL's zijn honorifics als awards, maar dat had heel lang geleden gekund, en was het vast toen fout in nu gecorrigeerd. 

We kunnen bij de scrum spreken over hoe die te regelen. Moet in de onto. We kunnen wat vals spelen door de acroniemen als waarden toe te voegen zodat de goed tonen. Of meer eerlijk de vollen namen invoeren en dan tonen wat lang na zijn naam. Er is vast een Fresnel oplossing waarmee de label van de honorific een sparql query naar zijn suffix is. Maar dan heeft DP en Wpedia geen suffix info voor honorifics.

De "awards" link moet ik maken als een eigen ontology uitbreiding met als waard een pagina. Of gewoon die negeren. Met als verhaal dat hij gewoon te veel awards heeft.

**Comment by AlexMekkering** [comment]
@LloydRutledge, do you want to leave this issue open until after ABI-Team-30? If so, could we also label it 'Post ABI-Team-30'?

**Comment by LloydRutledge** [comment]
Yes, but like the others I'll have to see in the coming months what purpose it still serves. Perhaps then I should only have issues that are direct bug fixes or specific new features.
